Demo/Demo/FeatureViewControllers/PaymentButtonCustomizationViewController.swift

```diff
@@ -0,0 +1,233 @@
+//
+//  PaymentButtonCustomizationViewController.swift
+//  Demo
+//
+//  Created by Jose Noriega on 25/05/2022.
+//
+
```

Comment:
```suggestion
```

Reply:
+1 Here we should always remove the Xcode generated header.

Reply:
Should we have a standard IDE template for new files? I sometimes forget to delete this, or other generated code, and also people tend to have their own formatting style that may look different to what the SDK implements

Reply:
That would be ideal. I'm unaware of a way to have a versioned template that Xcode can reference when generating a new file header, but there must be a way.

Removing the headers has been the easiest way so far, but I agree that it's a good idea to keep improving our workflows whenever possible.